features/theme.feature

```diff
@@ -335,27 +335,27 @@ Feature: Manage WordPress themes
 
   Scenario: Install and attempt to activate a child theme without its parent
     Given a WP install
-    And I run `wp theme install biker`
-    And I run `rm -rf wp-content/themes/jolene`
+    And I run `wp theme install buntu`
+    And I run `rm -rf wp-content/themes/stargazer`
 
-    When I try `wp theme activate biker`
+    When I try `wp theme activate buntu`
     Then STDERR should contain:
       """
-      Error: The parent theme is missing. Please install the "jolene" parent theme.
+      Error: The parent theme is missing. Please install the "stargazer" parent theme.
       """
     And STDOUT should be empty
     And the return code should be 1
 
   Scenario: List an active theme with its parent
     Given a WP install
-    And I run `wp theme install jolene`
-    And I run `wp theme install --activate biker`
+    And I run `wp theme install stargazer`
+    And I run `wp theme install --activate buntu`
 
     When I run `wp theme list --fields=name,status`
     Then STDOUT should be a table containing rows:
       | name          | status   |
-      | biker         | active   |
-      | jolene        | parent   |
+      | buntu         | active   |
+      | stargazer        | parent   |
```

Comment:
@wojsmol Can you just fix this alignment issue, please? Will be good for merging then.

Reply:
fixed in https://github.com/wp-cli/extension-command/pull/105/commits/63a25759f7e2a8f931069019fa74634e962c607a